osdep/macOS_swift_extensions.swift

```diff
@@ -56,3 +56,16 @@ extension NSScreen {
         }
     }
 }
+
+extension NSColor {
+
+    convenience init(hex: String) {
+        let int = Int(hex.dropFirst(), radix: 16)
```

Comment:
In swift Int is system bitness native 32bit or 64bit, so on desktop we should be OK with this (only 64bit macOS is supported, anyways).

We might want to switch to an unsigned integer later if that seems possible.